qcodes/tests/helpers/test_json_encoder.py

```diff
@@ -168,3 +168,25 @@ def test_numpy_encoder_examplemetadata():
         'myuserdict': {'a': 1}
     }
     assert metadata == data_dict
+
+
+def test_bytes():
+    e = NumpyJSONEncoder()
+    value = b"abc123"
```

Comment:
do we want a test for not str-able bytes that raises but not a recursion error?

Reply:
Looks like this is not actually an issue

```
IIn [20]: a ="☃️"

IIn [21]:

IIn [21]:

In [21]: c = bytes(a.encode("utf-8"))

In [22]: str(c)
Out[22]: "b'\\xe2\\x98\\x83\\xef\\xb8\\x8f'"
```

Will convert test to hypothesis to be sure